connectivity/FEATURE_BLE/libraries/TARGET_CORDIO/include/ble/internal/PalAttClientImpl.h

```diff
@@ -381,13 +381,11 @@ class PalAttClient : public interface::PalAttClient {
     struct PrepareWriteResponseConverter : ResponseConverter<ATTC_PREPARE_WRITE_RSP> {
         static AttPrepareWriteResponse convert(const attEvt_t* event)
         {
-            // WARNING: Not sure if correct, the stack erase the length parameter
             return AttPrepareWriteResponse(
                 event->handle,
-                to_uint16_t(event->pValue + 2),
-                // FIXME: the stack set the lenght to 0, the data won't be seen ...
+                0, /* offset is lost */
```

Comment:
Has what changed? The code you pointed to? Please be more direct.

Reply:
I was wondering if the cordio stack had changed the way it fills the `attEvt_t` data structure. 
Look like it has changed: 

https://github.com/ARMmbed/cordio-stack/blob/d205f85b10ab2c018617fda3bc18e0178e47b6f6/sw/stack/att/attc_write.c#L45-L60

It's reassuring to think this used to work before...

Reply:
Prepare writes? I doubt that. The conversion? Looks like it might have for the offset value but apparently not for length according to the old comment.